pygromos/files/blocks/coord_blocks.py

```diff
@@ -1,16 +1,18 @@
 import re
 from enum import Enum
-from typing import List, Iterable
+from typing import List
 from numbers import Number
 
 from pygromos.files.blocks._general_blocks import _generic_gromos_block, _generic_field, _iterable_gromos_block
-from pygromos.files.blocks._general_blocks import TITLE, TIMESTEP, TRAJ
+from pygromos.files.blocks._general_blocks import TITLE as default_TITLE
+from pygromos.files.blocks._general_blocks import TIMESTEP as default_TIMESTEP
+from pygromos.files.blocks._general_blocks import TRAJ as default_TRAJ
 from pygromos.files import blocks
 
 # forward declarations
-TITLE: TITLE = TITLE
-TIMESTEP: TIMESTEP = TIMESTEP
-TRAJ: TRAJ = TRAJ
+TITLE: default_TITLE = default_TITLE
+TIMESTEP: default_TIMESTEP = default_TIMESTEP
+TRAJ: default_TRAJ = default_TRAJ
```

Comment:
what is the difference of TITLE and default TITLE? its both class types just different naming

Reply:
rename to generic instead of default